README.md

```diff
@@ -211,32 +211,113 @@ This can be useful if using a column for doc references.
 
 ## doc references
 
+In the reference example below. A `property` on the json doc will be populated with the `doc` that matches the `WHERE` statement.
+The CSV example below using the reference will find the `doc` with `district_1` and create a `parent` property with the value of the `district_1` doc
+
 To reference other docs, replace the type suffix with a matching clause:
 
-	GET location:place WHERE external_id=COL_VAL
+	parent:place WHERE reference_id=COL_VAL
```

Comment:
I think it is easier to read the notation if you include the GET

eg `parent:GET place WHERE reference_id=COL_VAL`

Reply:
Having GET causes a problem. Not sure if it's a bug but it would set the referencedDoc to undefined. The current way the code works is without the GET.